README.md

```diff
@@ -161,10 +161,10 @@ Our [stability guide] helps to judge how much churn can be expected when dependi
 
 ### Download a Binary Release
 
-Using `cargo quickinstall`, one is able to fetch [binary releases][releases]. You can install it via `cargo install cargo-quickinstall`, assuming 
```

Comment:
Could you provide more context to help me understand why `quickinstall` is replaced?
To me it seems it's a viable alternative, and both will work equally well.

The way I understand it is that `quickinstall` will download binaries that it somehow obtained or built itself. To my mind, what `binstall` does is certainly preferable, even though it seems that some might still benefit from `quickinstall`.

To me it seems it would be better to mention `binstall` first, but also mention `quickinstall` and why some might want to use that instead.

Please note that my knowledge is definitely spotty here and includes a lot of guessing, so I'd be happy to be educated.

Reply:
> The way I understand it is that quickinstall will download binaries that it somehow obtained or built itself.

Quickinstall only uses binaries it built in its own repository, we have a mechanism for building popular crates (all binstall and quickinstall would HEAD an API to report usage).

Binstall can download from official releases and fallback to quickinstall if there's none official binary for this target, binstall also has a complex target fallback chain (e.g. x86_64-unknown-linux-gnu, fallback to musl).